components/developer/gdb/Makefile

```diff
@@ -20,24 +20,25 @@
 #
 
 #
-# Copyright 2017 Gary Mills
 # Copyright (c) 2011, 2014, Oracle and/or its affiliates. All rights reserved.
+# Copyright 2017 Gary Mills
+# Copyright 2019 Andreas Wacknitz
 #
 
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		gdb
 COMPONENT_VERSION=	7.10.1
-COMPONENT_REVISION=	4
+COMPONENT_REVISION=	5
+COMPONENT_SUMMARY=  GDB: The GNU Project Debugger
```

Comment:
Tab might do better.

Reply:
I have tried to fix the tabs, does it look better now?